docker/Dockerfile.agent

```diff
@@ -2,7 +2,7 @@ FROM python:3.9
 
 COPY . /src
 RUN mkdir -p /src/beamer/data/relayers
-COPY ./relayer/relayer-node16-*-x64 /src/beamer/data/relayers/
+COPY ./relayer-node16-*-x64 /src/beamer/data/relayers/
```

Comment:
How can this work? Does local invocation of `make container-image` still work?

Reply:
> Why would we ever run this locally?

I have been using this to test our Docker images. Far easier than to mess with github CI.
Also, it is not just about us -- this is intended to be usable by other developers too, and they may well expect this kind of thing to work. Think of the way that we are using Optimism's Docker infrastructure.
Anyone should be able to generate their own, possibly customised, Docker images. Why should we restrict this only to our CI?

Reply:
Ok, tested it again. works both locally and on ci.